uniffi_bindgen/src/bindings/python/templates/ErrorTemplate.py

```diff
@@ -13,6 +13,11 @@ def __str__(self):
             str(ctypes.cast(self.message, ctypes.c_char_p).value, "utf-8"),
         )
 
+class InternalError:
+    @staticmethod
+    def raise_err(code, message):
+        raise Exception("Unexpected error {}: {}".format(code, message))
```

Comment:
nit: please throw a `RuntimeException` here rather than the generic `Exception` base class.